unity/CITools/BuildDriver/Program.cs

```diff
@@ -86,14 +86,21 @@ public static int Main(string[] args)
             Description = "Produce zip artifacts. This is only used when building."
         };
 
+        var deployToPlayer= new Option<string>("--deploy-to-player")
+        {
+            DefaultValueFactory = (_) => string.Empty,
+            Description = "Copies the artifacts into a player build"
```

Comment:
If you wanted to get fancy you could add a custom validator here to ensure the provided path is a real one. Or for bonus points check to ensure that the provided directory will overwrite the correct files.

Reply:
I added some argument validation